signage:
  ros__parameters:
    emergency: true
    restart_engage: true
    door_close: true
    door_open: true
    engage: true
    thank_you: true
    in_emergency: true
    going_to_depart: true
    going_to_arrive: true
